src/test/java/org/folio/rest/impl/InvoiceLinesApiTest.java

```diff
@@ -100,6 +100,16 @@ public void getInvoicingInvoiceLinesByIdTest() throws Exception {
 
     logger.info(JsonObject.mapFrom(resp).encodePrettily());
     assertEquals(id, resp.getId());
+
+    // MODINVOICE-86 calculate the totals and if different from what was retrieved, write it back to storage
+    Double existingTotal = invoiceLinesList.getJsonArray("invoiceLines")
+      .getJsonObject(0)
+      .getDouble("total");
+    assertThat(existingTotal, equalTo(2.00d)); // outdated total in storage
+    double expectedTotal = 2.42d;
+    
+    final InvoiceLine updatedResponse = verifySuccessGet(INVOICE_LINES_PATH + "/" + id, InvoiceLine.class);
+    assertThat(updatedResponse.getTotal(), equalTo(expectedTotal)); // updated total after recalculating
```

Comment:
I think it's also worth checking that the correct value was sent with the PUT /invoice/invoice-lines request.